src/index.js

```diff
@@ -5,4 +5,28 @@ import './images/ExcursiOnward-logo.png'
 import Trips from './Trips';
 import Destination from './Destination'
 import Traveler from './Traveler'
-import Agency from './Agency'
\ No newline at end of file
+import Agency from './Agency'
+
+import {
+  getTraveler,
+  getTrips,
+  getDestinations,
+  addTrip,
+  updateTrip,
+  deleteTrip
+} from './utility.js';
+
+const dashboardGreeting = document.querySelector('.dashboard-greeting')
```

Comment:
Do you think you will be changing the dashboard greeting enough to make it a global variable?

Reply:
I always forget that query selectors can live in the function that updates them. Thanks for the reminder!